package.json

```diff
@@ -1,29 +1,29 @@
 {
   "name": "@nrwl/nx-source",
-  "version": "13.3.0-beta.2",
+  "version": "9999.0.2",
   "description": "Smart, Extensible Build Framework",
   "homepage": "https://nx.dev",
   "private": true,
   "scripts": {
     "build": "./scripts/package.sh --local",
     "commit": "git-cz",
     "check-commit": "node ./scripts/commit-lint.js",
-    "check-format": "nx format:check --all",
+    "check-format": "npx nx format:check --all",
     "check-imports": "node ./scripts/check-imports.js",
     "check-lock-files": "node ./scripts/check-lock-files.js",
-    "check-internal-links": "ts-node -P ./scripts/tsconfig.scripts.json ./scripts/documentation/internal-link-checker.ts",
-    "check-versions": "ts-node -P ./scripts/tsconfig.scripts.json ./scripts/check-versions.ts",
-    "check-documentation-map": "ts-node -P ./scripts/tsconfig.scripts.json ./scripts/documentation/map-link-checker.ts",
-    "update-nx-dev-documentation": "ts-node -P ./scripts/tsconfig.scripts.json ./scripts/documentation/nx-dev-release-content.ts",
+    "check-internal-links": "npx ts-node -P ./scripts/tsconfig.scripts.json ./scripts/documentation/internal-link-checker.ts",
```

Comment:
You shouldn't mix Yarn and npm, using `npx` here is unnecessary.

Reply:
@merceyz I agree. How about I include [yarn dlx](https://yarnpkg.com/cli/dlx) as a replacement for `npx`? Let me know if you think this is fine.

Reply:
Just remove `npx` and call `ts-node` directly as it was the case in the removed lines. When run through `yarn`, it will automatically check the locally installed packages for a corresponding binary.